### ISSUE TITLE
Wuhan issues review

### ISSUE THREAD

**Issue by anastassiavybornova** [uscuni/neatnet#137]
**singles**

***

![image](https://github.com/user-attachments/assets/e709add2-1447-42ca-bcdd-9d8b89d8ae3b)
![image](https://github.com/user-attachments/assets/ae010136-c48c-4e5a-b135-e45827a6b99b)

adding these just to give an example - there's quite a lot of "almost overlapping" carriageways like that mapped out in wuhan (my suspicion is also that those are the ones that made it necessary to increase the eps)

***

![image](https://github.com/user-attachments/assets/e345ef31-55d6-4699-b260-c3097fbd983e)

simplification didn't change that one (cf for Liège singles!) - but could we just blame that on notsogreat mapping..?

***

![image](https://github.com/user-attachments/assets/ee92edbd-78df-422e-83b4-0eed87a85e1d)
![image](https://github.com/user-attachments/assets/9ba05464-6307-473f-8a00-4298f44e8be4)

same here, nothing changed - is that ok?

***

**Comment by jGaboardi** [comment]
Yeah, we can close I think. But also funny timing that just now we have a [Wuhan failure in CI](https://github.com/uscuni/neatnet/actions/runs/15203862545/job/42762724093) from #216.
